dashboard/client/src/components/PlacementGroupTable.tsx

```diff
@@ -0,0 +1,155 @@
+import {
+  Box,
+  InputAdornment,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  TextField,
+  TextFieldProps,
+  Tooltip,
+} from "@material-ui/core";
+import Autocomplete from "@material-ui/lab/Autocomplete";
+import Pagination from "@material-ui/lab/Pagination";
+import React, { useState } from "react";
+import rowStyles from "../common/RowStyles";
+import { PlacementGroup } from "../type/placementGroup";
+import { useFilter } from "../util/hook";
+import StateCounter from "./StatesCounter";
+import { StatusChip } from "./StatusChip";
+
+const PlacementGroupTable = ({
+  placementGroups = [],
+  jobId = null,
+}: {
+  placementGroups: Array<PlacementGroup>;
+  jobId?: string | null;
+}) => {
+  const [pageNo, setPageNo] = useState(1);
+  const { changeFilter, filterFunc } = useFilter();
+  const [pageSize, setPageSize] = useState(10);
+  const placementGroupList = placementGroups.filter(filterFunc);
+  const list = placementGroupList.slice(
+    (pageNo - 1) * pageSize,
+    pageNo * pageSize,
+  );
+  const classes = rowStyles();
+
+  const columns = [
+    { label: "ID" },
+    { label: "Name" },
+    { label: "Job Id" },
+    { label: "State" },
+    { label: "Scheduling Detail" },
+  ];
+
+  return (
+    <React.Fragment>
```

Comment:
Let's use a div instead of fragment here. It makes sense to logically group these items together in a container.